engine/src/mac-dialog.mm

```diff
@@ -219,14 +219,28 @@ void MCPlatformBeginFolderDialog(MCPlatformWindowRef p_owner, MCStringRef p_titl
         [t_panel setMessage: [NSString stringWithMCStringRef: p_prompt]];
     }
     else
-        [t_panel setTitle: [NSString stringWithMCStringRef: p_prompt]];
+    {
+        extern uint4 MCmajorosversion;
+        if (MCmajorosversion >= 0x10B0)
+            [t_panel setMessage: [NSString stringWithMCStringRef: p_prompt]];
+        else
+            [t_panel setTitle: [NSString stringWithMCStringRef: p_prompt]];
+        
+    }
+        //[t_panel setTitle: [NSString stringWithMCStringRef: p_prompt]];
```

Comment:
You can remove the commented out line. Could you also add a comment before the initial clause of the 'if' here to explain what it is doing and why? (i.e. that if there is title and message set the both, otherwise if mac os < 10.11, use title; otherwise use message)